src/meta/protos/proto/table.proto

```diff
@@ -124,6 +124,8 @@ message TableMeta {
 
   // Comments of each field.
   repeated string field_comments = 25;
+
+  optional string storage_params = 26;
```

Comment:
Store our config in `string` is very easy to break.

Maybe we can use `StageStorage`?

https://github.com/datafuselabs/databend/blob/3b78959ca483cbd62abc322faadd1a0c59b2fead/src/meta/protos/proto/user.proto#L131

Visit @drmingdrmer for advice.

Reply:
Right, using a structured type would be better.